app/uk/gov/hmrc/rasapi/config/AppContext.scala

```diff
@@ -17,37 +17,33 @@
-  lazy val fileProcessingInternalServerErrorStatus: String = runModeConfiguration.getString("status.internal-server-error.csv").getOrElse("problem-getting-status")
-  lazy val internalServerErrorStatus: String = runModeConfiguration.getString("status.internal-server-error.api").getOrElse("INTERNAL_SERVER_ERROR")
-  lazy val removeChunksDataExerciseEnabled: Boolean = runModeConfiguration.getBoolean("remove-chunks-data-exercise.enabled").getOrElse(false)
-  lazy val apiV2_0Enabled: Boolean = runModeConfiguration.getBoolean("api-v2_0.enabled").getOrElse(false)
+class AppContext @Inject()(val servicesConfig: ServicesConfig) {
+  lazy val appName = servicesConfig.getString("appName")
+  lazy val appUrl = servicesConfig.getString("appUrl")
+  lazy val apiContext = servicesConfig.getString(s"api.context")
+  lazy val baseUrl = servicesConfig.getString("baseUrl")
+  lazy val apiStatus = servicesConfig.getString("api.status")
+  lazy val endpointsEnabled = servicesConfig.getBoolean("api.endpointsEnabled")
+  lazy val desAuthToken = servicesConfig.getString("desauthtoken")
+  lazy val desUrlHeaderEnv: String =  servicesConfig.getString("environment")
+  lazy val edhUrl: String = servicesConfig.getString("endpoints.edh.url")
+  lazy val resultsExpriyTime: Int = servicesConfig.getInt("results.expiry.time")
+  lazy val allowNoNextYearStatus: Boolean = servicesConfig.getBoolean("toggle-feature.allow-no-next-year-status")
+  lazy val allowDefaultRUK: Boolean = servicesConfig.getBoolean("toggle-feature.allow-default-ruk")
+  lazy val retryEnabled: Boolean = servicesConfig.getBoolean("toggle-feature.retry-enabled")
+  lazy val bulkRetryEnabled: Boolean = servicesConfig.getBoolean("toggle-feature.bulk-retry-enabled")
+  lazy val requestRetryLimit: Int = servicesConfig.getConfInt("request-retry-limit", 3)
+  lazy val retryDelay: Int = servicesConfig.getConfInt("retry-delay", 500)
+  lazy val deceasedStatus: String = servicesConfig.getString("status.deceased")
+  lazy val tooManyRequestsStatus: String = servicesConfig.getConfString("status.too-many-requests", "TOO_MANY_REQUESTS")
+  lazy val matchingFailedStatus: String = servicesConfig.getConfString("status.matching-failed.api", "STATUS_UNAVAILABLE")
+  lazy val serviceUnavailableStatus: String = servicesConfig.getConfString("status.service-unavailable", "SERVICE_UNAVAILABLE")
+  lazy val doNotReProcessStatus: String = servicesConfig.getConfString("status.do-not-re-process", "DO_NOT_RE_PROCESS")
+  lazy val fileProcessingMatchingFailedStatus: String = servicesConfig.getConfString("status.matching-failed.csv", "cannot_provide_status")
+  lazy val fileProcessingInternalServerErrorStatus: String = servicesConfig.getConfString("status.internal-server-error.csv", "problem-getting-status")
+  lazy val internalServerErrorStatus: String = servicesConfig.getConfString("status.internal-server-error.api", "INTERNAL_SERVER_ERROR")
```

Comment:
getConfString(value, default) isn't equivalent to getString(value).getOrElse(default) - getConfString prepends "microservice.services" to the value it's looking up, thus leaving the config like this would always return the default value

Reply:
as mentioned to Dan if you check over the config. I was happy to change these to getString but it would probably be best for someone who knows the service that understands why some of the defaults have been set and make a decision on what config can be changed and what can't.

Reply:
for example, these defaults were set before this change which was exactly the same as the config values set in app.conf which to mean seems like pointless defaults but that's not my call to make. The ones with getConfString defaults are set to the same as in the conf anyway so I didn't see it as an issue. Would be good to go through it and check over what defaults are actually needed.